.github/snapshot-changeset.md

```diff
@@ -9,6 +9,7 @@
 '@blocksuite/affine-components': patch
 '@blocksuite/affine-block-paragraph': patch
 '@blocksuite/affine-block-list': patch
+'@blocksuite/affine-widget-scroll-anchoring': patch
```

Comment:
Can you please also add other packages here?

Reply:
ok